debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerSnapshotChangeEventSource.java

```diff
@@ -44,7 +45,7 @@
     private final SqlServerConnectorConfig connectorConfig;
     private final SqlServerConnection jdbcConnection;
     private final SqlServerDatabaseSchema sqlServerDatabaseSchema;
-    private Map<TableId, SqlServerChangeTable> changeTables;
+    private Map<SqlServerTaskPartition, Map<TableId, SqlServerChangeTable>> partitionedChangeTables;
```

Comment:
Now it could be made final and initialized right away:
```java
private final Map<SqlServerTaskPartition, Map<TableId, SqlServerChangeTable>> partitionedChangeTables = new HashMap<>();
```
The `partitionedChangeTables` name may be confusing since the tables are not partitioned (like the change source). `changeTablesByPartition` might be a better alternative.